pre-sync/oci-image-verification/README.md

```diff
@@ -0,0 +1,139 @@
+# Sample OCI Image Signature Verification with Config Sync
+
+## Prerequisites
+
+CLI: [OpenSSL], [Cosign], [Gcloud], [Docker], [Kubectl], [Crane]
+
+Environment: GKE cluster, Google Cloud Artifact Registry repo
+
+## Build the webhook server
+
+```angular2html
+docker build -t gcr.io/<PROJECT_ID>/webhook-server:latest . && docker push gcr.io/<PROJECT_ID>/webhook-server:latest
+```
+
+## Create a namespace
+
+```angular2html
+kubectl create ns oci-webhook
+```
+
+```angular2html
+kubectl apply -f webhook-deployment.yaml
+```
+
+## Authentications
+
+### Cosign keys
+
+Generate cosign.key and cosign.pub
+
+```angular2html
+cosign generate-key-pair
+```
+
+Create on cluster secret:
+
+```angular2html
+kubectl create secret generic cosign-key --from-file=cosign.pub -n oci-webhook
```

Comment:
Should we change `oci-webhook` to <NAMESPACE> given that users are likely to choose their own namespaces?

Reply:
Added a comment about oci-webhook being a sample namespace.

Reply:
Namespace changed to signature-verification